src/NServiceBus.Core/OpenTelemetry/Metrics/Meters.cs

```diff
@@ -16,4 +16,10 @@ class Meters
 
     internal static readonly Counter<long> TotalFailures =
         NServiceBusMeter.CreateCounter<long>("nservicebus.messaging.failures", description: "Total number of messages processed unsuccessfully by the endpoint.");
+
+    internal static readonly Histogram<double> ProcessingTime =
+     NServiceBusMeter.CreateHistogram<double>("nservicebus.messaging.processingtime", "ms", "The time in milliseconds between when the message was pulled from the queue until processed by the endpoint.");
```

Comment:
What is the relationship between NServiceBusMeter.CreateHistogram and OpenTelemetry?

Reply:
See CreateHistogram:
https://learn.microsoft.com/en-us/dotnet/api/system.diagnostics.metrics.meter.createhistogram?view=net-8.0

NserviceBusMeter is of type Meter:
https://learn.microsoft.com/en-us/dotnet/api/system.diagnostics.metrics.meter?view=net-8.0

There is actually no direct reference to the OpenTelemetry SDK from Meter in .NET or from  NServiceBus, even though there is an `endpointConfiguration.EnableOpenTelemetry()` method. 

It was actually a great decision by the .NET team to keep system.diagnostics.metrics decoupled from OpenTelemetry, seeing as how many unified observability/telemetry tools have come before it.  There's no guarantee OpenTelemetry will be the last, but there has been a great deal more buying across the industry it seems.  That's all in the realm of interesting, but not important.

What is important is that Meter surfaces information in a way it can be consumed by otel and any proprietary tools or future standard tools. 

Sorry about mistakes, while on my phone.